Open Judge System/Web/OJS.Web/Areas/Administration/ViewModels/Contest/ChangeTimeForParticipantsViewModel.cs

```diff
@@ -37,11 +37,18 @@ public class ChangeTimeForParticipantsViewModel
         public int TimeInMinutes { get; set; }
 
         [Display(Name = "Participants_created_after", ResourceType = typeof(Resource))]
-        [Required(ErrorMessageResourceName = "Time_required_error", ErrorMessageResourceType = typeof(Resource))]
         public DateTime? ParticipantsCreatedAfterDateTime { get; set; }
 
         [Display(Name = "Participants_created_before", ResourceType = typeof(Resource))]
-        [Required(ErrorMessageResourceName = "Time_required_error", ErrorMessageResourceType = typeof(Resource))]
         public DateTime? ParticipantsCreatedBeforeDateTime { get; set; }
+
+        [UIHint("UsersSimpleComboBox")]
+        public string UserId { get; set; }
+
+        [Display(Name = "Change_time_by_time_interval", ResourceType = typeof(Resource))]
```

Comment:
Don't you think it is more robust to use `nameof(Resource,Change_time_by_time_interval)`?